homeassistant/components/rfxtrx/sensor.py

```diff
@@ -26,14 +26,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def _battery_convert(value):
```

Comment:
this change shouldnt be needed if the one test commented on below is updated

Reply:
I agree. on that. Even if it's not changed. I think they should remain internally marked. pylint errors in unit tests for usage of internal functions can imho be ignored.